apps/meteor/app/apps/server/converters/rooms.js

```diff
@@ -210,6 +211,17 @@ export class AppRoomsConverter {
 
 				return this.orch.getConverters().get('departments').convertById(departmentId);
 			},
+			closedBy: async (room) => {
+				const { closedBy } = room;
+
+				if (!closedBy) {
+					return undefined;
+				}
+
+				delete room.servedBy;
```

Comment:
closedBy and servedBy can hold diff data (ex, when a manager closes a conversation that an agent is serving)

Reply:
ops